File: Volume 11 (1100 - 1199)/1117/1117.rb
#!/usr/bin/env ruby

total = 0
i = 0
begin
    nota = gets.to_f
    if nota > 10 or nota < 0
        puts "nota invalida"
    else
        total += nota
        i += 1
    end
end while i < 2
puts "media = %.2f" % [total/2]

File: Volume 11 (1100 - 1199)/1117/1117.c
#include <stdio.h>
 
int main() {
    double n, total = 0, soma = 0;
    while (total != 2) {
        scanf("%lf", &n);
        if (n < 0 || n > 10) printf("nota invalida\n");
        else {
            soma += n;
            total++;
        }
    }
    printf("media = %.2lf\n", soma/2);
    return 0;
}